packages/app/src/pages/CustomizePracticePage.vue

```diff
@@ -35,25 +35,38 @@
         </q-step>
 
         <template v-slot:navigation>
-          <q-stepper-navigation class="justify-center">
+          <q-stepper-navigation class="flex justify-center q-gutter-sm">            
```

Comment:
Is this file change supposed to be a part of this PR?

Reply:
Oh no, it;s not supposed to be. I think I added that to my previous PR but forgot to push it!!

Reply:
Let's revert the changes from this PR then :) 

You can do so via git checkout master  packages/app/src/pages/CustomizePracticePage.vue